### PR TITLE
Feature/including

### PR DESCRIPTION
**Pull request by plally**

### CODE REVIEW

File: gamemode/load_modules.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
+local moduleDir = GM.FolderName.."/gamemode/modules/"
+
+local _, modules = file.Find( moduleDir.."*", "LUA" )
+
+for _, moduleName in pairs( modules ) do
+    local path = moduleDir..moduleName.."/"
+    local files, _ = file.Find( path.."*.lua", "LUA" )
```
Comment (samuelWilliams99) [inline gamemode/load_modules.lua]:
```suggestion
    local files, _ = file.Find( path .. "*.lua", "LUA" )
```

Comment (samuelWilliams99) [reply gamemode/load_modules.lua]: Do the rest as well pls

File: gamemode/load_modules.lua [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    
+    for _, file in pairs( files ) do
+        local fullPath = path..file
```
Comment (samuelWilliams99) [inline gamemode/load_modules.lua]: ew spaces

File: gamemode/init.lua [hunk 3]
```diff
@@ -1,17 +1,5 @@
...
-include( "config/drowning.lua" )
-AddCSLuaFile( "config/drowning.lua" )
-
--- Modules
-include( "modules/drowning/sv_drowning.lua" )
-AddCSLuaFile( "modules/drowning/cl_drowning.lua" )
-
-include( "modules/hunger/sv_hunger.lua" )
-
 resource.AddFile( "materials/icons/health.png" )
```
Comment (samuelWilliams99) [inline gamemode/init.lua:5]: Can we do something for automatically adding resource files?

Comment (plally) [reply gamemode/init.lua:5]: this can be done  in another pr

File: gamemode/load_modules.lua [hunk 4]
```diff
@@ -0,0 +1,24 @@
+local moduleDir = GM.FolderName .. "/gamemode/modules/"
+
+local _, modules = file.Find( moduleDir .. "*", "LUA" )
+
+for _, moduleName in pairs( modules ) do
+    local path = moduleDir .. moduleName .. "/"
```
Comment (samuelWilliams99) [inline gamemode/load_modules.lua:6]: Could be nice to add some prints in here, so it's clear what modules have been loaded successfully.
Something like
`print( "[RVR] Loading module " .. moduleName )`
and
`print( "[RVR] Finished loading modules" )`
at the end.

**Review by samuelWilliams99 — APPROVED** [review]
This looks ready to merge to me, following comment about prints would be nice, but not required (please do it though 😄 )

File: gamemode/load_modules.lua (hunk 4 above)
Comment (samuelWilliams99) [reply gamemode/load_modules.lua:6]: Could even put some information about which realms have been used in the print, like "Loaded shared module" if server and client side files were found!

**Pull request merged**